docs/static/azure-module.asciidoc

```diff
@@ -0,0 +1,458 @@
+the health of your infrastructure in real-time, accelerating root cause analysis
+and decreasing overall time to resolution. The Azure module helps you: 
+
+* Analyze infrastructure changes and authorization activity
+* Identify suspicious behaviors and potential malicious actors
+* Perform root-cause analysis by investigating user activity
+* Monitor and optimize your SQL DB deployments.
+
+NOTE: The Logstash Azure module is an
+https://www.elastic.co/products/x-pack[{xpack}] feature under the Basic License
+and is therefore free to use. Please contact
+mailto:monitor-azure@elastic.co[monitor-azure@elastic.co] for questions or more
+information.
+
+The Azure module uses the
+{logstash-ref}/plugins-inputs-azure_event_hubs.html[Logstash Azure Event Hubs
+plugin] to consume data from Azure Event Hubs. The module taps directly into the
+Azure dashboard, parses and indexes events into Elasticsearch, and installs a
+suite of {kib} dashboards to help you start exploring your data immediately.   
+
+[[azure-dashboards]]
+==== Dashboards
+
+These {kib} dashboards are available and ready for you to use. You can use them they are, or tailor them to meet your needs.
+
+===== Infrastructure activity monitoring
+
+* *Overview*. Top-level view into your Azure operations, including info about users, resource groups, service health, access, activities, and alerts. 
+
+* *Alerts*. Alert info, including activity, alert status (activated, resolved, succeeded), and alerts heatmap
+
+* *User Activity*. Info about system users, their activity, and requests.
+
+===== SQL database monitoring
+
+* *SQL DB Overview*. Top-level view into your SQL databases, including counts for databases, servers, resource groups, and subscriptions.
+
+* *SQL DB Database View*. Detailed info about each SQL database, including wait time, errors, DTU and storage utilization, size, and read and write input/output.
+
+* *SQL DB Queries*. Info about SQL database queries, including DTU Utilization, errors, and query duration and wait time. 
+
+==== Azure_event_hubs plugin
+
+The Azure module uses the `azure_event_hubs` plugin. Basic understanding of the
+plugin and options is helpful when you set up the Azure module. See
+{logstash-ref}/plugins-inputs-azure_event_hubs.html[azure_event_hubs plugin
+documentation] for more information about configurations and options. 
+
+[[azure-module-prereqs]]
+==== Prerequisites
+
+The Elastic Stack and Microsoft Azure Event Hubs are required for this module.
+
+[[azure-elk-prereqs]]
+===== Elastic prereqs
+
+Logstash, Elasticsearch, and Kibana should be installed and running. The
+products are https://www.elastic.co/downloads[available to download] and easy to
+install. 
+
+The Elastic Stack version 6.4 (or later) is required for this module. 
+
+NOTE: Logstash, Elasticsearch, and Kibana must run locally. You can also run 
+Elasticsearch, Kibana and Logstash on separate hosts to consume data from Azure.
```

Comment:
Seems like the Prereqs designation clarifies without adding additional words. I avoid extra words whenever possible because "the more you write, the less people read." 
If you feel strongly that there is risk of confusion, I will add.